features/networking/multus.feature

```diff
@@ -1133,19 +1133,18 @@ Feature: Multus-CNI related scenarios
   Scenario: Create pod with Multus ipvlan CNI plugin	
     # Make sure that the multus is enabled
     Given the multus is enabled on the cluster
-    And I store all worker nodes to the :nodes clipboard
     And the default interface on nodes is stored in the :default_interface clipboard
+    And evaluation of `node.name` is stored in the :target_node clipboard
```

Comment:
side-question:  I'm not sure if we always need to use a clipboard?  I guess if we implicitly change nodes in some steps, but if we don't change nodes can we use `<%= node.name %>` directly?

Does it work to do:
```
| ["spec"]["nodeName"]                                       | <%= node.name %>   |
```

Reply:
Yep, we can definitely use node.name here as the behavior is driven here on a single node only. We generally save ```cb``` in case there is too much detours on various nodes which is clearly not  in this case. Thanks!